test/integration/auth.spec.ts

```diff
@@ -164,6 +164,136 @@ describe('admin.auth', () => {
       });
   });
 
+  describe('getUsers()', () => {
+    /**
+     * Filters a list of object to another list of objects that only contains
+     * the uid, email, and phoneNumber fields. Works with at least UserRecord
+     * and UserImportRecord instances.
+     */
+    function mapUserRecordsToUidEmailPhones(values: Array<{ uid: string, email?: string, phoneNumber?: string}>) {
+      return values.map((ur) => ({ uid: ur.uid, email: ur.email, phoneNumber: ur.phoneNumber }));
+    }
+
+    const testUser1 = { uid: 'uid1', email: 'user1@example.com', phoneNumber: '+15555550001' };
+    const testUser2 = { uid: 'uid2', email: 'user2@example.com', phoneNumber: '+15555550002' };
+    const testUser3 = { uid: 'uid3', email: 'user3@example.com', phoneNumber: '+15555550003' };
+    const usersToCreate = [ testUser1, testUser2, testUser3 ];
+
+    const importUser1: admin.auth.UserImportRecord = {
```

Comment:
Add a comment why this is here. I assume it's because you wanted a test user with a specific provider config.

Reply:
Done.